### ISSUE TITLE
[Screen reader - ADF - Azure data factory] Screen reader doesn't announce state of Azure data factory control.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3848]
**User Experience:**
Screen reader user won't get state of any control if screen reader doesn't provides state for that control and also he/she might get confused for such control. Not only this they won't be able to proceed further because of such confusion and also won't be able to proceed further. 

**Test Environment:**
Version: 20H2 (OS Build: 19042.508)
Platform: Windows
Application Version: 1.16.0-dev
Build Number: 20200921.1
Screen reader: Narrator, NVDA

**Note:** User credentials should NOT be included in the bug.

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it. 
**Repro Steps:​**
- Launch Microsoft Azure Storage Explorer app.
- Navigate and select visual icon control Add data factory control at left panel.
- Verify state of Add data factory control by using screen reader.
**Actual Result:**
Screen reader doesn't announce state of 'Add data factory' control . It only announce Add data factory button' which might create confusion for the screen reader user as they won't get an idea of button functionality.
**Expected Result:**
Screen reader should announce the state(Expand/Collapsed) of 'Add data factory' control along with dialog window . It should announce Add data factory button collapsed/expanded so that screen reader user could get an idea of button functionality.
**Note:**
NVDA also doesn't announce the state of Add data factory button.
**MAS Reference:**
[MAS 1.3.1 - Info and Relationships](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7d)
**Reference Links:**

- **[Accessibility Insights!** ](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906834622&sdata=0WbVE4dkxMrMm9bbxE1O5xshTfo1%2B0frnzlbmWVeHTs%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.
- **External Bug Process:** If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906839611&sdata=D5fhtDtz6f9O9hGT6oPoowXq0L/TCfaejH73thazOvA%3D&reserved=0). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
- **Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process related queries.**
[MAS1.3.1Azure data factory.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641617/MAS1.3.1Azure.data.factory.zip)
